spigot/pom.xml

```
@@ -13,7 +13,7 @@
 
     <description>Decays the leaves of the Fungi trees (Nether trees)</description>
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>17</java.version>
```

```suggestion
        <java.version>1.8</java.version>
```
You are welcome to build with a more recent version of Java, but at the end this plugin still targets Minecraft 1.16, which requires only Java 8*.

_*and given the small scale of this plugin, it is still affordable to support an outdated Java version._

---

I was unable to actually build it without changing the java version, it wouldn't compile. I had attempted to reset my IDE before doing that, somehow now, after changing it back to 1.8, it's working so I legitimately have no idea what was wrong there.